tests/features/contact_form/submit_contact_form.feature

```diff
@@ -43,6 +43,16 @@ Feature: Submit the contact form
     And I should see the following success messages:
       | success messages                                              |
       | Your message has been submitted. Thank you for your feedback. |
+    When I click the link matching the "#https?://[^/].*?/contact_form/\d{4}-\d{2}/logo(_\d+)?.png#" pattern from the email sent to "digit-joinup@ec.europa.eu"
```

Comment:
Let's try to rewrite this in a way that is understandable by the business stakeholders and the functional team. Regular expressions are a bit too technical :sweat_smile: